### PR TITLE
fix: add development environment variable

### PR DESCRIPTION
**Pull request by httpsmenahassan**
Resolves https://github.com/openedx/frontend-app-learner-dashboard/issues/210

### CODE REVIEW

**Review by arbrandes — APPROVED** [review]
Great, thanks!

File: .env.development [hunk 1]
```diff
@@ -48,3 +48,4 @@ ACCOUNT_PROFILE_URL='http://localhost:1995'
 ENABLE_NOTICES=''
 CAREER_LINK_URL=''
 OPTIMIZELY_FULL_STACK_SDK_KEY=''
+EXPERIMENT_08_23_VAN_PAINTED_DOOR=true
```
Comment (arbrandes) [inline .env.development:51]: Actually, mind setting this to `false`?  We don't actually need the experiment to run, just for the variable to be set.

**Review by arbrandes — CHANGES_REQUESTED** [review]
Just one thing, as commented above.

File: .env.development (hunk 1 above)
Comment (httpsmenahassan) [reply .env.development]: @arbrandes all set! changed to false

**Review by arbrandes — APPROVED** [review]

**Comment by brian-smith-tcril** [comment]
@arbrandes I just tried this locally with `EXPERIMENT_08_23_VAN_PAINTED_DOOR=false` and I ran into the same `TypeError`

I also got the `TypeError` when changing this line
https://github.com/openedx/frontend-app-learner-dashboard/blob/82ff0d7ddb95e316ef6c0251de49bafebe0b93d9/src/containers/WidgetContainers/AppWrapper/index.jsx#L9
to be
```jsx
if (false) {
```

I'm doing a bit more digging now

**Comment by brian-smith-tcril** [comment]
I think https://github.com/openedx/frontend-app-learner-dashboard/pull/218 should do the trick

**Comment by httpsmenahassan** [comment]
@brian-smith-tcril thanks for digging into this, Brian! I'll take a look at your PR

**Comment by brian-smith-tcril** [comment]
To follow up on this, I tried `EXPERIMENT_08_23_VAN_PAINTED_DOOR=false` again today and no longer could reproduce the error. I set a breakpoint

https://github.com/openedx/frontend-app-learner-dashboard/blob/82ff0d7ddb95e316ef6c0251de49bafebe0b93d9/src/containers/WidgetContainers/AppWrapper/index.jsx#L9

and saw `process.env.EXPERIMENT_08_23_VAN_PAINTED_DOOR` evaluating to `true`

I'm not sure what magic I did on Friday to get strings to parse as bools, but it doesn't seem to be working now (all non-empty strings are being treated as true)

When I set `EXPERIMENT_08_23_VAN_PAINTED_DOOR=''` instead, `process.env.EXPERIMENT_08_23_VAN_PAINTED_DOOR` evaluated to `false` and I got to the same `TypeError`

**Comment by justinhynes** [comment]
I'll be bringing this PR our next team refinement session for prioritization.

**Comment by jsnwesson** [comment]
Hey @brian-smith-tcril and @arbrandes, Team Aperture was originally assigned `needs maintainer attention` for both this PR and #218 , but according to Brian's comment above, #218 would make this PR redundant. If that is the case, should we only review #218 (as in only have that PR reviewed by the team responsible for the experiment)?

**Comment by brian-smith-tcril** [comment]
@jsnwesson I think reviewing #218 first makes sense. If that solution is seen as problematic for some reason then this PR should be merged as a workaround until a better solution is agreed upon.

**Comment by arbrandes** [comment]
#218 was merged.  Do we still need this?

**Comment by brian-smith-tcril** [comment]
@arbrandes

> Do we still need this?

nope! #218 should do the trick!

**Comment by httpsmenahassan** [comment]
Thanks, everyone! I'll go ahead and close this.

**Pull request closed without merging**